realm/realm-annotations-processor/src/test/resources/some/test/AllTypes.java

```diff
@@ -70,6 +70,7 @@
     private RealmList<Date> columnDateList;
     private RealmList<Decimal128> columnDecimal128List;
     private RealmList<ObjectId> columnObjectIdList;
+    private final RealmList<AllTypes> columnFinalRealmList = new RealmList<AllTypes>();
```

Comment:
There are files in `realm-annotation-processor/src/test/resources/io/realm` tracking the expected output of running Realm on the test classes. I think you would need to update those too.

Reply:
Thanks for pointing it out, have updated these resources.